src/thunderbots/software/ai/world/robot.h

```diff
@@ -220,13 +263,13 @@ class Robot
     // The id of this robot
     unsigned int id_;
     // The current position of the robot, with coordinates in metres
```

Comment:
Probably want to update these comments to reflect the fact that these variables now store the robot history. Something like:
``` suggestion
     // All previous positions of the robot, with the most recent position at the back of the queue, coordinates in meters
```

Reply:
Fixed